src/toast/Toast.tsx

```diff
@@ -0,0 +1,55 @@
+import "./_toast.scss";
+
+import React, {useEffect} from "react";
+import ReactDOM from "react-dom";
+import classNames from "classnames";
+
+import {useToast} from "./util/toastHooks";
+
+function Toast() {
+  const {
+    toastState: {
+      isDisplayed,
+      data: {autoClose, timeout, content, mode, customClassName}
+    },
+    dispatchToastAction
+  } = useToast();
+  let toastRootNode = document.querySelector("#toast-root");
```

Comment:
Let's add a `toastRootId` prop
```suggestion
  let toastRootNode = document.querySelector(toastRootId || "#toast-root");
```